docs/Privacy/Private-Transactions/Creating-Sending-Private-Transactions.md

```diff
@@ -3,11 +3,34 @@ description: Creating and sending private transactions
 
 # Creating and Sending Private Transactions 
 
-To create and send private transactions use one of: 
+Create and send private transactions using: 
 
-* [web3.js-eea client library](eeajs.md)
+* [web3.js-eea client library](eeajs.md) or [web3j client library](https://github.com/web3j/web3j)
 * [`eea_sendTransaction` with EthSigner](https://docs.ethsigner.pegasys.tech/en/latest/Using-EthSigner/Using-EthSigner/) 
+* [`eea_sendRawTransaction`](../../Reference/Pantheon-API-Methods.md#eea_sendrawtransaction) 
 
 !!! note
     Private transactions either deploy contracts or call contract functions. 
-    Ether transfer transactions cannot be private. 
\ No newline at end of file
+    Ether transfer transactions cannot be private. 
+    
+## Methods for Private Transactions
+
+A private transaction creates a [Privacy Marker Transaction](../Private-Transaction-Processing.md) as well the private transaction itself. 
```

Comment:
as well "as". Alternatively "A private transaction creates a Privacy Marker Transaction in addition to the private transaction itself.

Reply:
Fixed